docs/integrations-guides/rollup-guides/integrations-overview.md

```diff
@@ -77,70 +87,64 @@ is disabled, and state roots cannot be challenged. This means the sequencer can
 post whatever state roots they want to the bridge contract and potentially steal
 funds.
 
-## L2 Inbox Certificate Verification Strategy (M1) {#M1}
+## Cert Timeliness Verification
```

Comment:
You're calling it "timeliness" here instead of punctuality. Any reason?

Reply:
nope, changed: [92e7eb1](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/92e7eb1fc69d470faadb47589269e13b757187bc)